planet/index_demos.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

```

Comment:
this file is generated automatically, but i will update the template as well so no action needed